bucket/kops.json

```
@@ -0,0 +1,21 @@
+{
+    "version": "1.11.0",
+    "description": "Kubernetes Operations (kops) - Production Grade K8s Installation, Upgrades, and Management.",
+    "homepage": "https://github.com/kubernetes/kops/",
+    "license": "Apache License 2.0",
```

Invalid SPDX identifier

```suggestion
    "license": "Apache-2.0",
```